### ISSUE TITLE
Workarounds for webpack import on vue-cli

### ISSUE THREAD

**Issue by jennydaman** [dattn/dnd-grid#24]
The code segment in [README.md](https://github.com/dattn/dnd-grid#how-to-import-using-es6-import) do not work for projects managed with [vue-cli](https://github.com/vuejs/vue-cli), which depends on [webpack](https://webpack.js.org/). 

```javascript
// import Container and Box components
import { Container, Box } from '@dattn/dnd-grid'
// minimal css for the components to work properly
import '@dattn/dnd-grid/dist/index.css'
```

`Container` and `Box` are undefined. I was able to work around this problem with the code below:

```javascript
import dnd from "@dattn/dnd-grid";
import "@dattn/dnd-grid/dist/dnd-grid.css";

export default {
  name: "app",
  components: {
    DndGridContainer: dnd.Container,
    DndGridBox: dnd.Box
  }
```

Has anyone found a better way to do this?

**Comment by dattn** [comment]
This commit should fix you issue: https://github.com/dattn/dnd-grid/commit/7b9e32c9215bc183c4d9571de25900b82a7c9465
I will release a new version soon

**Comment by dattn** [comment]
Fixed in v0.0.13 release

**Comment by jennydaman** [comment]
Thank you!
